### PR TITLE
Add createdDateTime and lastUpdated fields

### PR DESCRIPTION
**Pull request by LeoKHoward**
# Motivation and Context
New fields in CASE_UPDATED and CASE_CREATED events

# What has changed
New fields added

# How to test?
Build other repos on card and run ATs

# Links
https://trello.com/c/n6FwePKv/1052-waiting-on-action-processor-casecreated-caseupdated-events-should-contain-the-createddatetime-field-5

https://collaborate2.ons.gov.uk/confluence/pages/viewpage.action?spaceKey=CATD&title=Response+Management+Event+Dictionary

### CODE REVIEW

**Comment by LeoKHoward** [comment]
[caseCreated/caseUpdated events should contain the createdDateTime field (5)](https://trello.com/c/n6FwePKv/1052-casecreated-caseupdated-events-should-contain-the-createddatetime-field-5)

**Comment by LeoKHoward** [comment]
[caseCreated/caseUpdated events should contain the createdDateTime field (5)](https://trello.com/c/n6FwePKv/1052-casecreated-caseupdated-events-should-contain-the-createddatetime-field-5)

File: src/main/java/uk/gov/ons/census/action/model/entity/Case.java [hunk 1]
```diff
@@ -1,9 +1,14 @@
 package uk.gov.ons.census.action.model.entity;
 
 import com.vladmihalcea.hibernate.type.json.JsonBinaryType;
+import java.time.OffsetDateTime;
 import java.util.UUID;
 import javax.persistence.*;
+import javax.persistence.Entity;
```
Comment (NickGrantX) [inline src/main/java/uk/gov/ons/census/action/model/entity/Case.java]: something has gone wrong with code formatting here

File: src/main/java/uk/gov/ons/census/action/model/entity/Case.java [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
+import javax.persistence.Index;
+import javax.persistence.Table;
 import lombok.Data;
+import org.hibernate.annotations.*;
```
Comment (NickGrantX) [inline src/main/java/uk/gov/ons/census/action/model/entity/Case.java]: more code formatting problems

**Review by NickGrantX — CHANGES_REQUESTED** [review]
Something weird going on with your code formatter. Are you running the `fmt:fmt` Maven plugin before commit & push?

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
Test failures in the travis build

**Review by AdamHawtin — APPROVED** [review]
👍 Building fine locally for me now

**Review by NickGrantX — APPROVED** [review]

**Pull request merged**